src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/ParametersValidatorUtilTest.java

```diff
@@ -54,6 +54,22 @@ public void stringIsNull() throws JobParametersInvalidException {
         ParametersValidatorUtil.checkIsNotNullOrEmptyString(null, JOB_PARAMETER_NAME);
     }
 
+    @Test(expected = JobParametersInvalidException.class)
+    public void stringWithAsciiCharacter() throws JobParametersInvalidException {
```

Comment:
Maybe this method should be called `stringWithNonAsciiCharacter` (and the one below should be renamed too), but in any case a string like this should be supported.